docs/homebrewPostgresInstructions.md

```diff
@@ -0,0 +1,12 @@
+# Homebrew - (Mac Only)
+## Installing PostgreSQL
+
+1. Install using hombrew: `brew install postgres`
+2. Add a local user (we recommend creating as a superuser to keep setup simple):  `create user postgres --superuser`
```

Comment:
Doesn't the user called `postgres` exist by default? Either way I recommend calling it something different. 

It's worth also recommending use of Postgres.app, which I find way easier to work with. 


https://postgresapp.com/